OracleJava/16/Dockerfile

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2020,2021 Oracle and/or its affiliates. All rights reserved.
```

Comment:
You should stop including "All rights reserved." in new copyright statements as we no longer include it. However, you do not need to change old statements. As this is a new file, I think you should remove it from this file (and from any other new file in which it appears).